website_rma/views/website_templates.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="contact_us_request_rma" name="Request RMA" inherit_id="website.contactus">
+        <xpath expr="//div[@name='mail_button']" position="after">
```

Comment:
don't work with website_crm module, it removes the mail_button div.

Reply:
I just tested and they seem to work fine altogether

Reply:
Ok, @jesusVMayor I just reproduced the issue :) @ernestotejeda You should take an alternative hook

Reply:
@jesusVMayor , @chienandalu it's done